docs/configuration-reference/components/aws-ebs-csi-driver.md

```diff
@@ -26,6 +26,9 @@ Sample config:
 # aws-ebs-csi-driver.lokocfg
```

Comment:
this commit message seems a bit weird. How about

`docs: update docs for aws-ebs-csi-driver`